src/routes/docs/tutorials/apple/step-6/+page.markdoc

```diff
@@ -7,6 +7,66 @@ step: 6
 
 # Authentication {% #authentication %}
 
+In order to allow creating OAuth sessions, the following URL scheme must be added to your **Info.plist** file.
```

Comment:
I don't think we are actually using OAuth in this flow. This means we should be able to omit the changes to the `Info.plist` altogether.